gulpfile.js

```diff
@@ -0,0 +1,144 @@
+// Gulpfile.js
+// Require Plugin Dependencies
+// 19 FEB 2015
+var gulp        = require('gulp'),
+    rename      = require('gulp-rename'),
+    concat      = require('gulp-concat');
+// -----------------------------------------------------------------------------
+// Objects Path
+var htdocs = {
+  js: {
+    custom          : './htdocs/js/',
+    vendor          : './htdocs/js/vendor'
+  },
+  sass: {
+    vendor          : './htdocs/sass/vendor/'
+  },
+  framework: {
+    zurbFoundation  : './htdocs/sass/vendor/zurb_foundation',
+    componentMixins : './htdocs/sass/vendor/zurb_foundation/foundation/components'
+  }
+};
+
+var packages = {
+  // Feature detection library
+  features: [
+    './vendor/package/modernizr/modernizr.js'
+  ],
+
+  // JavaScript libraries
+  jsLibraries: [
+    './vendor/package/jquery/dist/jquery.min.js',
+    './vendor/package/enquire/dist/enquire.min.js'
+  ],
+
+  // Normalize
+  normalize: [
+    './vendor/package/normalize-scss/_normalize.scss'
+  ],
+
+  // Function and mixin library
+  sukiyaki: [
+    './vendor/package/sukiyaki/scss/**/*'
+  ],
+
+  // Mayer reset
+  css_reset: [
+    './vendor/package/meyer-reset/stylesheets/_meyer-reset.scss'
+  ],
+
+  // Zurb Foundation 5
+  framework: {
+    import_components: [
+      './vendor/package/foundation/scss/foundation.scss'
+    ],
+    core: [
+      './vendor/package/foundation/scss/foundation/*'
+    ],
+    mixin: [
+      './vendor/package/foundation/scss/foundation/components/_global.scss',
+      './vendor/package/foundation/scss/foundation/components/_grid.scss'
+    ]
+  }
+};
+
+// -----------------------------------------------------------------------------
+// Gulp Tasks
+
+// Feature detection
+gulp.task('features', function() {
+  gulp.src(packages.features)
```

Comment:
@prakarangs you should always `return` the gulp task. see http://stackoverflow.com/a/21700789

```js
return gulp.src()
  .pipe();
```